src/main/java/org/openstreetmap/atlas/tags/MinHeightTag.java

```diff
@@ -0,0 +1,35 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.Optional;
+
+import org.openstreetmap.atlas.geography.Altitude;
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.Tag.Validation;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+import org.openstreetmap.atlas.tags.annotations.validation.DoubleValidator;
+
+/**
+ * OSM min_height tag
+ *
+ * @author ajayaswal
+ */
+@Tag(value = Validation.DOUBLE, taginfo = "https://taginfo.openstreetmap.org/keys/min_height#values", osm = "https://wiki.openstreetmap.org/wiki/Key:min_height")
+public interface MinHeightTag
+{
+    @TagKey
+    String KEY = "min_height";
+
+    DoubleValidator validator = new DoubleValidator();
```

Comment:
Because this is getting used only in a static method below do you think it will be a good idea to have it as
``` private static final DoubleValidator doubleValidator = new DoubleValidator()``` ?

Reply:
Its an interface, so can't do private.